website/themes/beastie/layouts/partials/sidenav.html

```diff
@@ -62,16 +62,16 @@
   <li>
     <a href={{ "releases" | absLangURL }}>{{ i18n "releases" }}</a>
     <ul>
-      <li>{{ i18n "productionRelease" }}: <br /><a href={{ index .data.releases "u-rel131-announce" }}>{{ index .data.releases "rel131-current" }}</a></li>
-      <li>{{ i18n "productionRelease" }}: <br /><a href={{ index .data.releases "u-rel130-announce" }}>{{ index .data.releases "rel130-current" }}</a></li>
       <li>{{ i18n "productionRelease" }}: <br /><a href={{ index .data.releases "u-rel123-announce" }}>{{ index .data.releases "rel123-current" }}</a></li>
+      <li>{{ i18n "productionRelease" }}: <br /><a href={{ index .data.releases "u-rel131-announce" }}>{{ index .data.releases "rel131-current" }}</a></li>
     </ul>
   </li>
   <li>
     <a href={{ "snapshots" | absLangURL }}>{{ i18n "snapshotReleases" }}</a>
       <ul>
         <li>{{ i18n "upcomingRelease" }}:<br /><a href={{ index .data.releases "u-betarel-schedule" }}>{{ index .data.releases "betarel-current" }}</a></li>
         <li>{{ i18n "upcomingRelease" }}:<br /><a href={{ index .data.releases "u-betarel2-schedule" }}>{{ index .data.releases "betarel2-current" }}</a></li>
+        <li>{{ i18n "upcomingRelease" }}:<br /><a href={{ index .data.releases "u-betarel3-schedule" }}>{{ index .data.releases "betarel3-current" }}</a></li>
```

Comment:
Is this correct?

Reply:
… I can check the sanity for myself, with a local build of the site. 

<https://docs.freebsd.org/en/books/fdp-primer/doc-build/#documentation-build-example-server>, I guess.

Reply:
Checked. Screenshot at <https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=266167#c6>.